contribs/gmf/src/controllers/desktop.scss

```diff
@@ -790,7 +790,7 @@ gmf-featurestyle {
   }
```

Comment:
Should be removed :-)

Reply:
Is it not used in the non-canvas interface (desktop interface) ?

Reply:
Oups your right :-)